### ISSUE TITLE
Large description is large

### ISSUE THREAD

**Issue by vegasgeek** [norcross/gf-tooltips#7]
Have you considered forcing a max height/width and adding scroll bars to the pop-up description box for when a description gets really big? For example, I'm trying to use the tooltips to display the Terms and Conditions for a form. Currently the block of text runs off the top of the screen when I tried it.

**Comment by norcross** [comment]
no, no I hadn't. I was thinking since they were 'tips', they'd be short. I could add something, though. pull request?
